application/src/main/resources/thingsboard.yml

```diff
@@ -1242,8 +1242,10 @@ transport:
   snmp:
     # Enable/disable SNMP transport protocol
     enabled: "${SNMP_ENABLED:true}"
+    # Snmp bind address
+    bind_address: "${SNMP_BIND_ADDRESS:0.0.0.0}"
     # Snmp bind port
```

Comment:
Please update the comment:
`# SNMP bind port. Zero (random) by default. When using SNMP TRAPs - make sure to specify some static value, e.g. 1620`
Also update `# Snmp bind address` to `# SNMP bind address`